### PR TITLE
PIMS-76 Updating documentation

### PR DESCRIPTION
**Pull request by Fosol**
Adding database design documentation.
Updating DevOps documentation.
Updating architecture documentation.

### CODE REVIEW

File: database/mssql/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ MSSQL_SA_PASSWORD=<password>
 MSSQL_PID=Developer
 ```
 
-Enter the `password` you want to secure the DB with.
+Enter the `password` you want to secure the DB with. This password needs to be appropriately
```
Comment (devinleighsmith) [inline database/mssql/README.md]: needs to be appropriately...

File: docs/ARCHITECTURE.md [hunk 2]
```diff
@@ -1,70 +1,33 @@
...
-## Architectural Design
-All components of the solution are configured for Docker containers to run independently and therefore scallable.
+![solution architecture](./images/arch-01.png)
 
-### APP
-The APP frontend will be a React application that will provide the user interface.  It will provide authentication/authorization with KeyCloak (Using Open ID Connect).  To communicate with the API it will proxy requests that begin with `/api` to the configured API host.
+### Membership
 
-It will be written in React and have a Node backend.
+PIMS access requires a validated account to access. A user with one of the following account; IDIR, BCeID, GitHub or a custom.
```
Comment (devinleighsmith) [inline docs/ARCHITECTURE.md]: one of the following account(s)

also are we planning on allowing github or custom accounts access to PIMS?

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Property Inventory Management System (PIMS)
+
+The PIMS solution will provide a geo-spatial inventory of properties to assist Strategic Real Estate Services branch of Real Property Division to manage and oversee the disposal of assets surplus to government, optimize the benefits to the government and citizens of BC related to the management of public real estate assets, and be a trusted source for accurate information for Government owned titled property.
+
+## Description
+
+Presently PIMS is a frontend web application that integrates with a backend API
```
Comment (devinleighsmith) [inline docs/README.md]: maybe cut this line.

File: database/mssql/README.md (hunk 1 above)
Comment (Fosol) [reply database/mssql/README.md]: I've udpated.

File: docs/README.md (hunk 3 above)
Comment (Fosol) [reply docs/README.md]: Cut.

File: docs/ARCHITECTURE.md (hunk 2 above)
Comment (Fosol) [reply docs/ARCHITECTURE.md]: I've changed the text to say optional.

File: docs/ARCHITECTURE.md [hunk 4]
```diff
@@ -1,70 +1,33 @@
...
+### Membership
 
-It will be written in React and have a Node backend.
+PIMS access requires a validated account to access. A user with one of the following account types; IDIR, BCeID (optionally we may also support GitHub or a custom account).
 
-### API
+### APP Architecture
 
-The API will be RESTful design, providing authentication/authorization with KeyCloak (Using JWT Bearer tokens).
+The front-end application is a Node React app. It is hosted by Nginx and the React app is built for a production release. It is developed with TypeScript to improve standards. It uses KeyCloak for authentication. The JWT token will contain authorization information and allows for communication with the API.
```
Comment (asanchezr) [inline docs/ARCHITECTURE.md]: I'd remove "is a Node React app". It is just a React app

File: docs/ARCHITECTURE.md [hunk 5]
```diff
@@ -1,70 +1,33 @@
...
-| ----------- | -------- | ------ | -------- | -------- |
-| Id          | Int      |        | \*       | Identity |
-| Lat         | Float    |        | \*       |
-| Lng         | Float    |        | \*       |
-| Note        | String   | 2000   |
-| OwnerId     | Guid     |        | \*       |
-| CreatedOn   | DateTime |        | \*       |
-| UpdatedOn   | DateTime |
-| UpdatedById | Guid     |
+The DevOps platform that support CI/CD along with the Operations platform environments are hosted with OpenShift. Additionally KeyCloak which provides authentication and authorization is hosted within OpenShift.
```
Comment (asanchezr) [inline docs/ARCHITECTURE.md]: small typo: "The DevOps platform that support(s) ..."  <- _Note the missing (s)_

File: docs/ARCHITECTURE.md [hunk 6]
```diff
@@ -1,70 +1,33 @@
...
 
-#### User
+The PIMS solution will be hosted within the OpenShift environment provided by the Lab ([here](https://console.pathfinder.gov.bc.ca:8443/console/projects)).
```
Comment (asanchezr) [inline docs/ARCHITECTURE.md]: "... provided by the (Exchange) Lab ..."

File: docs/ARCHITECTURE.md (hunk 4 above)
Comment (Fosol) [reply docs/ARCHITECTURE.md]: Fixed

File: docs/ARCHITECTURE.md (hunk 6 above)
Comment (Fosol) [reply docs/ARCHITECTURE.md]: Fixed

File: docs/ARCHITECTURE.md (hunk 5 above)
Comment (Fosol) [reply docs/ARCHITECTURE.md]: fixed

**Pull request merged**